content/research/machine-learning/_index.md

```diff
@@ -28,6 +28,10 @@ Machine learning (ML) algorithms, and more recently deep learning (DL) methods,
 
 {{% intro "/research/machine-learning/historical_virtual_control_groups.md" %}}
 
+## [Kinodata-3D](/projects/kinodata-3d/)
```

Comment:
can you also add the project under

- structural kinase work
- and kinoml (?)

and add a full new kinase ml related category and then add yours and Paula's project under their?